web-storybook/.storybook/preview.jsx

```diff
@@ -31,14 +38,20 @@ const StorybookI18nProvider = ({ children }) => {
 
 export const decorators = [
   Story => (
-    <StorybookI18nProvider>
-      <ThemeProvider injectFonts>
-        <CssBaseline />
-        <Router>
-          <Story />
-        </Router>
-      </ThemeProvider>
-    </StorybookI18nProvider>
+    <QueryClientProvider client={reactQueryClient}>
+      <AuthApolloProvider apolloClientFactory={apolloClientFactory}>
```

Comment:
not sure if we should have storybook depend on those providers, which are really marketplace related
Instead if we want to keep the UserAccountSettings story, we should probably refactor it to not use those providers directly but rather provide what we need as props (from `useEmailConfirmationData`)